### PR TITLE
added various templated websites

### LINKED ISSUE #2427: [Site Request] mangaproz
**Name of the website**
mangaproz

**Website urls **
- Site: https://mangaproz.com/
- Manga List: https://mangaproz.com/manga/
- Manga example: https://mangaproz.com/manga/nito-no-taidana-isekai-shoukougun/
- Chapter online viewer example: https://mangaproz.com/dff6486846/

**Language**
Arabic

### LINKED ISSUE #2500: [Site Request] 6mh7.com
**Name of the website**
6mh7

**Website urls (examples below)**
- Site: http://www.6mh7.com/
- Manga List: http://www.6mh7.com/rank/1-1.html
- Manga example: http://www.6mh7.com/22541/
- Chapter online viewer example: http://www.6mh7.com/22541/1109364.html

**Languages**
Chinese

**Additional details**
Add any other context details that you may have found (like template to reuse)

### LINKED ISSUE #2503: [Site Request] mangaproz
**Name of the website**
 mangaproz.com

**Website urls (examples below)**
- Site: https://mangaproz.com/
- Manga List: https://mangaproz.com/manga
- Manga example: https://mangaproz.com/manga/infinite-apostles-and-twelve-war-girls/
- Chapter online viewer example: https://mangaproz.com/infinostles-51/

**Languages**
arabic

### PR DESCRIPTION
**Pull request by 27136773**
References: dazedcat19/FMD2#154
References: dazedcat19/FMD2#155

### CODE REVIEW

**Comment by 27136773** [comment]
## Manual Regression Tests for SinMH based Websites

|                        | Copy & Paste | Get Mangas | Get Chapters | Get Images |
| ------------- | :------------: | :-----------: | :------------: | :----------: |
| **manhuagui**  | ✅ | ❌ | ✅ | ✅ |
| **jpm1234**      | ✅ | ✅ | ✅ | ✅ |
| **wuqimh**       | ✅ | ✅ | ✅ | ✅ |
| **manhuabei**  | ✅ | ✅ | ✅ | ✅ |
| **manhuafen**  | ✅ | ✅ | ✅ | ✅ |
| **manhuaniu**  | ✅ | ✅ | ✅ | ✅ |
| **gufengmh8** | ✅ | ✅ | ✅ | ✅ |

_ManHuaGui has disabled manga list scraper, to prevent IP ban_

File: src/web/mjs/connectors/SixMH7.mjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        super.id = '6mh7';
+        super.label = '6漫画 (6mh7)';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'http://www.6mh7.com';
+
+        this.path = '/sort/1-%PAGE%.html';
+        this.queryManga = 'div.cy_main div.cy_info div.cy_title h1';
+        this.queryMangasPageCount = 'div.NewPages ul li:last-of-type a';
+        this.queryMangas = 'div.cy_list_mh ul li.title a';
+        this.queryChapters = 'ul[id^="mh-chapter-list"] li a';
```
Comment (Robonau) [inline src/web/mjs/connectors/SixMH7.mjs]: this doesn't get all chapters, just the ones that are loaded on page

Comment (27136773) [reply src/web/mjs/connectors/SixMH7.mjs]: Good find
Fixed script to get all chapters

File: src/web/mjs/connectors/SixMH7.mjs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'http://www.6mh7.com';
+
+        this.path = '/sort/1-%PAGE%.html';
+        this.queryManga = 'div.cy_main div.cy_info div.cy_title h1';
+        this.queryMangasPageCount = 'div.NewPages ul li:last-of-type a';
+        this.queryMangas = 'div.cy_list_mh ul li.title a';
+        this.queryChaptersScript = `
+            new Promise((resolve, reject) => {
+                charpterMore(null);
```
Comment (Sheepux) [inline src/web/mjs/connectors/SixMH7.mjs:18]: charpterMore ? Really ?

Comment (ronny1982) [reply src/web/mjs/connectors/SixMH7.mjs:18]: I guess this has to be called to load the additional chapters in the browser window

**Review by Sheepux — APPROVED** [review]
/LGTM

**Pull request merged**